### ISSUE TITLE
Code review

### ISSUE THREAD

**Issue by tshemsedinov** [GIroSTarK/What-To-Do-List#1]
1. Never compare with logical values like here https://github.com/GIroSTarK/What-To-Do-List/blob/6d856b34f3277273109a3f75ce138fc7e1586982/app.js#L31 and here https://github.com/GIroSTarK/What-To-Do-List/blob/6d856b34f3277273109a3f75ce138fc7e1586982/app.js#L35 you can have just logical expression: `if (!task.classList.contains("completed") )`
2. Code duplication can be combined: https://github.com/GIroSTarK/What-To-Do-List/blob/6d856b34f3277273109a3f75ce138fc7e1586982/app.js#L30-L38
3. No tests
4. Very short and primitive project, typical to-do list can be found anywhere in the internet

**Comment by GIroSTarK** [comment]
I understood the first two points. And I also understand that similar to-do
lists can be found on the Internet, but I made my project myself, thinking
about the design and functions that were not available on the Internet. For
example, the function of adding a task to favorites is my idea, since I
have not seen anything similar on the Internet. All other functionality was
also implemented by myself. I realize that my project may be simple enough,
but I tried to refactor all the anti-patterns I know. I also made 28
commits, which showed in detail how my project was implemented and how much
code I refactored. Some students did only a few commits. As for the tests,
I know that there are people in my group who didn't do the tests but have
90+ points. Maybe my project is not worth 90+, but I think I deserve at
least 85 points.

сб, 10 черв. 2023 р. о 16:36 Timur Shemsedinov ***@***.***>
пише:
